packages/common/src/index.ts

```diff
@@ -23,6 +23,8 @@ export * from "./request";
 export * from "./surveys";
 export * from "./ArcGISContextManager";
 export * from "./ArcGISContext";
+export * from "./models";
+export * from "./projects";
```

Comment:
Can we put this in alphabetical order or grouped?

Reply:
I've significantly re-structured things and I'll open a new PR - I think it adds better/consistent organization